yarn.lock

```diff
@@ -2,443 +2,251 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.16.7":
-  "integrity" "sha512-iAXqUn8IIeBTNd72xsFlgaXHkMBMt6y4HJp1tIaK465CWLT/fG1aqB7ykr95gHHmlBdGbFeWWfyB4NJJ0nmeIg=="
-  "resolved" "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.16.7.tgz"
-  "version" "7.16.7"
-  dependencies:
-    "@babel/highlight" "^7.16.7"
-
 "@babel/code-frame@7.12.11":
```

Comment:
Dunno what's with the changes here but we'll remove one of the lockfiles after you decide on which one should stay.